### PR TITLE
Move results exploration to Running computations

### LINKED ISSUE #248: 🧪 TEST: Checklist for basics section and main page
List of things to check for the basics section of the tutorial:

- [ ] Add a comment in the "AiiDA basics" section to mention that it is taken from the basic aiida tutorial (even if just for us to remember in the future)
- [ ] search for "this tutorial" in the section copied from the docs and replace with the appropriate section name
- [x] replace link to (missing) appendix for computer setup with a link to the aiida-core docs after aiidateam/aiida-core#4216 is merged
- [X] ~~try to find a place where to mention groups~~ - This has been moved into the "Working with data and querying section" by @chrisjsewell in #231.
- [ ] try to find a place where to show the MC provenance explorer
- [X] ~~Move Outputs subsection to "Running computations" hands on.~~ See #251 
- [ ] Consider addressing #218 on pausing in basics (or running computations) section.
- [ ] See if we can wrap text in table columns to make sure tables fit the page width.
- [x] Add hands-on section for install once this is merged:
https://github.com/aiidateam/aiida-core/pull/4216#pullrequestreview-443055002

### PR DESCRIPTION
**Pull request by mbercx**
Moves the commands for exploring the outputs of the `PwCalculation` to the running computations hands on.

### CODE REVIEW

**Review by chrisjsewell — APPROVED** [review]

File: docs/pages/2020_Intro_Week/sections/running.rst [hunk 1]
```diff
@@ -589,16 +589,70 @@ Finally, let's label this calculation as a restarted one and submit the new calc
...
+
+.. tip::
 
-    calculation = load_node(<pk>)
-    calculation.res.energy
+    Filter the lines containing the string 'Fermi', e.g. using ``grep``, to isolate the relevant lines.
+
+The results of calculations are stored in two ways: ``Dict`` objects are stored in the database, which makes querying them very convenient, whereas ``ArrayData`` objects are stored on the disk.
+Once more, use the command ``verdi data array show <IDENTIFIER>`` to determine the Fermi energy obtained from calculation you ran.
+This time you will need to use the identifier of the output ``ArrayData`` of the calculation, with link name ``output_trajectory_array``.
```
Comment (ramirezfranciscof) [inline docs/pages/2020_Intro_Week/sections/running.rst:631]: Ok, so I think the `ArrayData` called `output_trajectory_array` is now a more specific `TrajectoryData` called `output_trajectory`. Also, the Fermi energy asked here is just what is reported as the "energy", right?

**Review by ramirezfranciscof — CHANGES_REQUESTED** [review]
So, I found a couple of minor issues. Could you change those and the code in line 586 (`builder` should be `restart_builder`)? This is an older typo so I cannot point it out in the reviewed code.

    .. code:: python

        restart_builder.metadata.label = 'Restart from PwCalculation<{}>'.format(failed_calculation.pk)
        calculation = submit(builder)

**Comment by ramirezfranciscof** [comment]
Wooops, I didn't realize it was already merged. I could open an issue for these.

**Comment by chrisjsewell** [comment]
Yep too slow 😂 pr to master now

**Comment by ramirezfranciscof** [comment]
Ok, tomorrow I'll make the PR with these fixes and the workchain one.

**Comment by mbercx** [comment]
Don't worry, I'll add these notes to a PR I'm working on now. I'll already merge it into `master`, but you can check it tomorrow morning.

File: docs/pages/2020_Intro_Week/sections/running.rst (hunk 1 above)
Comment (mbercx) [reply docs/pages/2020_Intro_Week/sections/running.rst:631]: > Ok, so I think the `ArrayData` called `output_trajectory_array` is now a more specific `TrajectoryData` called `output_trajectory`. 

Yes. This was also mentioned by @sphuber in #236. Unfortunately, we run an `scf` calculation here, so the `TrajectoryData` is not very interesting. I'll mention this and have them run a `vc-relax` as a final exercise.

> Also, the Fermi energy asked here is just what is reported as the "energy", right?

The Fermi energy is indeed not present in this calculation. I've adjusted this to just looking for the total energy (`energy`). 

Seems moving this section here caused more problems than I thought... Sorry about that.

**Pull request merged**